docs/specification.md

```diff
@@ -442,6 +443,56 @@ Parameter (in Michelson):
 - Fails with `FA2_INSUFFICIENT_BALANCE` if the wallet associated with the given address
 does not have enough tokens to burn.
 
+### **transfer_contract_tokens**
+
+```haskell
+type TokenId = Natural
+
+data TransferDestination = TransferDestination
+  { to_ :: Address
+  , token_id :: TokenId
+  , nat :: Natural
+  }
+
+data TransferParam = TransferParam
+  { from_ :: Address
+  , txs :: [TransferDestination]
+  }
+
+type TransferParams = [TransferParam]
+
+data Parameter proposalMetadata
+  = Transfer Address TransferParams
```

Comment:
```suggestion
  = TransferContractTokens Address TransferParams
```
Just renaming suggestion for consistent.


From what I understand, when this entrypoint is called, it delegates this call to another FA2 contract `transfer` entrypoint. Is this correct, Ivan?

Reply:
> Just renaming suggestion for consistent.

Oh, I forgot to rename it.

> From what I understand, when this entrypoint is called, it delegates this call to another FA2 contract transfer entrypoint. Is this correct, Ivan?

Yes, exactly. The list of `operation`s it returns should contain one `TRANSFER_TOKENS` operation that calls `transfer` entrypoint.

Reply:
Done, renamed to `Transfer_contract_tokens`.